testfiles/disabled-line.txt

```diff
@@ -1,3 +1,3 @@
 hello
-	world // editorconfig-checker-disable-line
+  world // editorconfig-checker-disable-line
```

Comment:
Why the indentation (from tabs to spaces) changed here?

Reply:
Please look at the first commit.

Let me know if I have to explain more

Reply:
Yes, I saw. But this change is unrelated to the feature this PR is trying to add, and could be made in a separated PR. 
I tried both with and without this change, and tests are passing both ways, as you explained it is because the code was already valid, so there was nothing (no errors) to ignore.

What confused me is that this PR is changing behavior close to `editorconfig-checker-disable-line` implementation, which made me think, `editorconfig-checker-disable-line` doesn't work anymore as intended.

Reply:
I can split the PR, no problem. We can talk about each of them separately.

Reply:
A reply was provided here

https://github.com/editorconfig-checker/editorconfig-checker/pull/363#issuecomment-2322607033

It's OK